README.md

```diff
@@ -165,4 +167,6 @@ The `result` object will contain (at most) the following structure:
 }
 ```
 
-The `options` contains the configuration object passed to `typeset`; this can be useful for passing other data along or for identifying which `typeset()` call is associated with this `callback` call (in case you use the same `callback` function for more than one `typeset()`).
+If the `errors` array is non-empty, the Promise will reject.
```

Comment:
How about:

    the Promise will reject, and be passed the `errors` array.

Reply:
Sounds good.